utils/value.go

```diff
@@ -46,9 +46,9 @@ type Rand interface {
 }
 
 // Testing returns true when you are running in test suite.
+// Deprecated: this is in the standard library now.
 func Testing() bool {
-	// TODO switch to official testing.Testing method when we are on go 1.21
-	return flag.Lookup("test.v") != nil
+	return testing.Testing()
 }
```

Comment:
I doubt this is being used by external users. Seems like it is only used internally in `utils/test_detector/main.go`

Not sure how we go about deleting unused but public code. (we are pre v1 so maybe find to delete)

Reply:
I doubt people are upgrading every single week to the new version. Unless we want to deprecate this now and remove in 1-2 months (or never remove it), I doubt it makes a difference if we wait a week to remove.

Safest is just to deprecate. But I would hope that we could just remove.

Reply:
Not too involved in rdk releases nowadays

Reply:
will leave as-is and make it a problem for future me